numenc_module.pyi

```diff
@@ -0,0 +1,25 @@
+# stubs for the IDE to provide autocompletion.
```

Comment:
(Comment C) Please rename the module to "numenc.pyi", "_module" suffix is non-informative.

Reply:
It's a convention for modules created with the Python C API to have the suffix "-module", but I agree with you it's unnecessary. Will remove it.